modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,127 @@
+To ensure that different storage strategies use the same persistent volume data structure, subpaths of the mounted persistent volumes that reference user-defined persistent volumes are prefixed with `_<workspace-ID>_` or `__<persistent-volume-name>__`. 
+See xref:how-subpaths-are-used-in-persistent-volumes_{context}[].
 
-* Subpaths of the mounted Physical persistent volumes that reference user-defined PVCs are prefixed with `_<workspace-ID>_` or `__<PVC-name>__`. This ensures that the same PV data structure is configured with different PVC strategies. For details, see xref:how-subpaths-are-used-in-pvcs_{context}[].
+When the user deletes a workspace, the workspace engine deletes all workspace persistent volumes.
 
-The `unique` PVC strategy is suitable for larger multi-node clusters with a lesser amount of users. Since this strategy operates with separate PVCs for each volume in a workspace, vastly more PVCs are created.
+Persistent volumes (PV) provisioning::
+This is the strategy creating the highest volumes count.
 
 
-[id="how-subpaths-are-used-in-pvcs_{context}"]
-== How subpaths are used in PVCs
+[id="how-subpaths-are-used-in-persistent-volumes_{context}"]
+== How subpaths are used in persistent volumes
 
 Subpaths illustrate the folder hierarchy in the Persistent Volumes (PV).
 
+[subs="+quotes,+attributes"]
 ----
-/pv0001
-  /workspaceID1
-  /workspaceID2
-  /workspaceIDn
+/__<pv0001>__
+  /__<workspaceID1>__
+  /__<workspaceID2>__
+  /__<workspaceIDn>__
     /che-logs
     /projects
-    /<volume1>
-    /<volume2>
-    /<User-defined PVC name 1 | volume 3>
+    /__<volume1>__
+    /__<volume2>__
+    /__<user-defined-volume-name-1>__
+    /__<user-defined-volume-name-2>__
+    /__<user-defined-volume-name-3|volume3>__
     ...
 ----
 
-When a user defines volumes for components in the devfile, all components that define the volume of the same name will be backed by the same directory in the PV as `__<PV-name>__`, `__<workspace-ID>__, or `__<original-PVC-name>__`. Each component can have this location mounted on a different path in its containers.
-
+When a user defines volumes for components in the devfile, all components that define the volume of the same name will be backed by the same directory in the persistent volume as `__<persistent-volume-name>__`, `__<workspace-ID>__`, or `__<original-volume-name>__`.
+Each component can have this location mounted on a different path in its containers.
 
 .Example
-Using the `common` PVC strategy, user-defined PVCs are replaced with subpaths on the common PVC. When the user references a volume as `my-volume`, it is mounted in the common-pvc with the `/workspace-id/my-volume` subpath.
+With the `common` storage strategy, the workspace engine replaces user-defined volumes with subpaths on the common persistent volume.
+When the user references the `__<my-volume>__` volume, the workspace engine mounts it on the common persistent volume with the `/__<workspace-id>__/__<my-volume>__` subpath.
```

Comment:
this section should be part of the 'common' strategy imo

Reply:
The common strategy already has it, let's remove it.